pkgs/development/tools/build-managers/gradle/tests.nix

```diff
@@ -0,0 +1,35 @@
+{ runCommand, testers, jdk11, jdk17, stdenv }:
+{ gradle, gradleWithToolchains ? null }:
+
+{
+  version = testers.testVersion {
+    package = gradle;
+  };
```

Comment:
`testVersion` checks the output for [`package.version`](https://github.com/NixOS/nixpkgs/blob/3cf3fef37260eb9c9cbf00096c48064f11dc83a9/pkgs/build-support/testers/default.nix#L61), which is Gradle’s version, not the JDK version, right?

Have not attempted run the test yet since it builds JDK, but it seems to me that it should fail…

Reply:
Yes, it's the other test below that one that checks the JDK versions when using toolchains. iirc i implemented this test because it's something the nixpkgs 
manual recommends as a trivial "can this thing even run" test.

Reply:
(As such it's really only about the Gradle version, not the JDK)